scss/bootstrap/overrides/style/_buttons.scss

```diff
@@ -60,39 +60,39 @@
 .btn-primary {
 
   &:hover {
-    background-color: $brand-primary-dark;
+    background-color: darken($btn-primary-bg, 7%);
   }
 }
 
 .btn-success {
   color: $gray-darker;
 
   &:hover {
-    background-color: $brand-success-dark;
-    color: $crm-white;
+    background-color: darken($btn-success-bg, 7%);
```

Comment:
this '7%' looks to me that should be in some variable, it's everywhere

Reply:
Ok I will make related changes